scout/resources/variant_evaluation_terms.json

```diff
@@ -0,0 +1,270 @@
+                "evidence": ["allele_count"]
+            },
+            {
+                "key": 2,
+                "label": "Suspected in affected",
+                "description": "Variant is suspected to be mosaic in a affected sample.",
+                "evidence": ["allele_count"]
+            },
+            {
+                "key": 3,
+                "label": "Confirmed in parent",
+                "description": "Variant is confirmed to be mosaic in a parent sample.",
+                "evidence": ["allele_count"]
+            },
+            {
+                "key": 4,
+                "label": "Confirmed in affected",
+                "description": "Variant is confirmed to be mosaic in a affected sample.",
+                "evidence": ["allele_count"]
+            },
+            {
+                "key": 5,
+                "label": "Not evident in parental reads",
+                "description": "Variant was inspected for mosaicism, but not seen in reads from parental samples.",
+                "evidence": ["allele_count"]
+            }
+        ]
+    },
+    {
+        "track": ["cancer"],
+        "category": "cancer_tier",
+        "terms": [
+            {
+                "key": "1A",
+                "label": "Tier IA",
+                "description": "Strong Clinical Significance. Biomarkers in FDA or guidlines that predict response, resistance to therapy, diagnosis or prognosis to specific tumor type.",
+                "label_class": "danger"
+            },
+            {
+                "key": "1B",
+                "label": "Tier IB",
+                "description": "Potential Clinical Significance Biomarkers in well-powered, concenus affirmed studies that predict response, resistance to therapy, diagnostic or prognostic significance to specific tumor type.",
+                "label_class": "danger"
+            },
+            {
+                "key": "2C",
+                "label": "Tier IIC",
+                "description": "Biomarkers in FDA or guidlines that predict response, resistance to therapy,to a different tumor type; are diagnostic or prognostic for multiple small studies; or serve as study inclusion criteria.",
+                "label_class": "warning"
+            },
+            {
+                "key": "2D",
+                "label": "Tier IID",
+                "description": "Biomarkers that show plausible therapeutic significance based on preclinical studies, may assist diagnosis or prognosis based on small reports.",
+                "label_class": "warning"
+            },
+            {
+                "key": "3",
+                "label": "Tier III",
+                "description": "Variant of Unknown Clinical Significance-Not observed in the population, nor in tumor databases.No convincing published evidence of cancer association.",
+                "label_class": "primary"
+            },
+            {
+                "key": "4",
```

Comment:
If the key is an integer crashes cancer variants list view. Also if you select this custom tier scout cant display that it is selcted in cancer variant view.

`{ "key": 6, "label": "Tier achtung", "description": "Wir haben nur ein stuhl", "label_class": "default" }`

Reply:
Very nice that you found this error, I'll fix!